### ISSUE TITLE
Suggestions for new sounds

### ISSUE THREAD

**Issue by yashashebbarabailu** [ashutoshgngwr/noice#534]
If you haven't found it already. I found a source for ambient sounds which might be helpful for adding more sounds. There are many free sound here. More noice please.

[Royalty free sounds](https://freetousesounds.bandcamp.com/)

**Comment by ashutoshgngwr** [comment]
Adding more sounds without a considerable number of requests for specific sounds will make the app bloated with useless sounds. Closing this for now. Let me know if you have any specific requests!

**Comment by yashashebbarabailu** [comment]
I get it. I will try to choose some specific sounds from that.

**Comment by yashashebbarabailu** [comment]
[Abandoned Courtyard Ambience! Leaking Water Pipe & Church Bells](https://freetousesounds.bandcamp.com/album/abandoned-courtyard-ambience-leaking-water-pipe-church-bells)

[Underwater Ambience! The Hidden Sounds](https://freetousesounds.bandcamp.com/album/underwater-ambience-the-hidden-sounds)

[Pumpjack Pumping Sound Effects! California, USA](https://freetousesounds.bandcamp.com/track/pumpjack-pumping-sound-effects-california-usa)

[Car Interior Sound Effects! Taxi Istanbul](https://freetousesounds.bandcamp.com/album/car-interior-sound-effects-taxi-istanbul)

[Whipping Wind Sounds Under Motorbike Helmet](https://freetousesounds.bandcamp.com/album/whipping-wind-sounds-under-motorbike-helmet)

I hope you find this interesting.

**Comment by VasilisKos** [comment]
> Adding more sounds without a considerable number of requests for specific sounds will make the app bloated with useless sounds. Closing this for now. Let me know if you have any specific requests!

I agree that the app should not be overbloated with useless sounds, like not natural ones.

**Comment by PanderMusubi** [comment]
Softly and rhythmically crashing of waves is missing in this app. One for waves at sea and one for waves in the surf on the beach. I used these from other apps in the past a lot and those where the best to fall asleep with, for me at least.

The current seaside sound in this app is more of a washing, splashing, dripping nature and therefore less relaxing and more motivating to have a pee.

If the requested waves are welcomed, I can search for usable source.

**Comment by ashutoshgngwr** [comment]
> If the requested waves are welcomed, I can search for usable source.

Please do. Perhaps the BBC sound effects have something? See #361.

**Comment by laralem** [comment]
Anyone agrees Noice should have a really good [wind chimes](https://en.wikipedia.org/wiki/Wind_chime) sound? Add it or, to avoid bloat, at least substitute "Wind in chimes of shells" already in Noice. Although that sound is also nice, I was expecting a more relaxing sound and without background noise. I've found 2 examples:
- https://freesound.org/people/giddster/sounds/437337/ CC0 license, almost perfect wind chimes -almost perfect because it seems it was touched by hands and not by the wind, according to sound and author description
- https://freesound.org/people/Littleboot/sounds/196712/ CC0 license, clean sound although it is and sounds artificial

By the way, new sound request should be made here or I should have opened a new "issue"?
